webapp/pages/profile/_id/_slug.vue

```diff
@@ -168,7 +168,7 @@
               tag="h5"
               color="soft"
             >
-              Wer folgt {{ user.name | truncate(15) }}?
+              Wer folgt {{ userName | truncate(15) }}?
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-19T11:40:42Z" title="Friday, April 19th 2019, 1:40:42 pm +02:00">Apr 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -167,7 +167,7 @@
               tag="h5"
               color="soft"
             >
-              Wer folgt {{ user.name | truncate(15) }}?
+              Wer folgt {{ userName(user.name,15) }}?
```


```suggestion
              Wer folgt {{ truncateUserName(user.name,15) }}?
```
If find this more telling. The method does not return the userName, it's truncating it. (+ having a user-specific default)

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-04-24T15:50:44Z" title="Wednesday, April 24th 2019, 5:50:44 pm +02:00">Apr 24, 2019</time>_

---


resorve by seperation